palette/src/gradient.rs

```diff
@@ -17,9 +17,9 @@ use Mix;
 ///It's used to smoothly transition between a series of colors, that can be
 ///either evenly spaced or have customized positions. The gradient is
 ///continuous between the control points, but it's possible to iterate over a
-///number of evenly spaced points using the `take` method. Any point outside
-///the domain of the gradient will have the same color as the closest control
-///point.
+///number of evenly spaced points using the `take` method. The `take` method is
+///now inclusive of both ends of the gradient. Any point outside the domain of
+///the gradient will have the same color as the closest control point.
```

Comment:
The "The `take` method is now inclusive of both ends of the gradient" part kind of reads like a release notes entry. I think we can safely skip that sentence.